src/main/java/org/broadinstitute/hellbender/utils/recalibration/RecalibrationArgumentCollection.java

```diff
@@ -99,11 +100,11 @@
     @Argument(fullName = "preserve_qscores_less_than", shortName = "preserveQ", doc = "Don't recalibrate bases with quality scores less than this threshold (with -" + StandardArgumentDefinitions.BQSR_TABLE_SHORT_NAME + ")", optional = true)
     public int PRESERVE_QSCORES_LESS_THAN = QualityUtils.MIN_USABLE_Q_SCORE;
 
-    @Hidden
+    @Advanced
     @Argument(fullName = "enableBAQ", shortName = "enableBAQ", doc = "do BAQ correction")
     public boolean enableBAQ = false;
 
-    @Hidden
+    @Advanced
```

Comment:
Add javadoc explaining why these two arguments are `@Advanced` and not turned on by default.